rule_packages/c/Language2.json

```diff
@@ -0,0 +1,45 @@
+{
+  "MISRA-C-2012": {
+    "DIR-4-2": {
+      "properties": {
+        "obligation": "advisory"
+      },
+      "queries": [
+        {
+          "description": "",
+          "kind": "problem",
+          "name": "All usage of assembly language should be documented",
+          "precision": "very-high",
+          "severity": "error",
```

Comment:
should these (this and RULE-1-4) be warning? they arent necessarily undefined behaviour level, just unpredicatable?

Reply:
Agree!

Reply:
I think you missed this one, still is error